dhis-2/dhis-web/dhis-web-portal/pom.xml

```diff
@@ -1,57 +1,106 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-  <parent>
-    <groupId>org.hisp.dhis</groupId>
-    <artifactId>dhis-web</artifactId>
-    <version>2.40-SNAPSHOT</version>
-  </parent>
+    <parent>
+        <groupId>org.hisp.dhis</groupId>
+        <artifactId>dhis-web</artifactId>
+        <version>2.40-SNAPSHOT</version>
+    </parent>
 
-  <artifactId>dhis-web-portal</artifactId>
-  <packaging>war</packaging>
-  <name>DHIS Web Portal</name>
+    <artifactId>dhis-web-portal</artifactId>
+    <packaging>war</packaging>
+    <name>DHIS Web Portal</name>
 
-  <build>
-    <finalName>dhis</finalName>
-  </build>
+    <build>
+        <finalName>dhis</finalName>
+        <plugins>
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <version>${jib.version}</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>com.google.cloud.tools</groupId>
+                        <artifactId>jib-ownership-extension-maven</artifactId>
+                        <version>0.1.0</version>
+                    </dependency>
+                </dependencies>
+                <configuration>
+                    <container>
+                        <environment>
+                            <!-- default DHIS2 web application context to / but allow for customization -->
+                            <JAVA_TOOL_OPTIONS>-Dcontext.path=""</JAVA_TOOL_OPTIONS>
```

Comment:
Is this still overridable with an environment variable at runtime (not build time)?  It was previously and is used by `d2 cluster` and potentially other places

Reply:
Yes, its part of the container environment. So `JAVA_TOOL_OPTIONS` can be set in a docker compose or on the docker CLI as environment flag 😄

Reply:
But that changed from the previous env var CATALINA_OPTIONS which would make this a breaking change right ?

Reply:
Adapted to set CATALINA_OPTS to `/` by default but allow it to be customized as done in d2 https://github.com/dhis2/docker-compose/blob/9dfd4c3a63ac91b9912f430d6caf15ffdc2f1fe7/cluster/docker-compose.yml#L12